platformio.ini

```diff
@@ -524,6 +524,29 @@ board_build.f_flash = 80000000L
 board_build.flash_mode = qio
 monitor_filters = esp32_exception_decoder
 
+[env:esp32s3dev_16MB_PSRAM_opi]
```

Comment:
see my first posting on how to simplify the section, by leveraging on `extends`.